tools/deployment/ort_test.py

```diff
@@ -0,0 +1,225 @@
+import argparse
+import os.path as osp
+import warnings
+
+import mmcv
+import numpy as np
+import onnxruntime as ort
+import torch
+from mmcv import Config, DictAction
+from mmcv.parallel import MMDataParallel
+
+from mmdet.apis import single_gpu_test
+from mmdet.core import bbox2result
+from mmdet.datasets import (build_dataloader, build_dataset,
+                            replace_ImageToTensor)
+from mmdet.models import BaseDetector
+
+
+class ONNXRuntimeDetector(BaseDetector):
```

Comment:
Suggest to move it to `mmdet.core.export`. Thus the downstream repo could also benefit from this module.

Reply:
No problem